### ISSUE TITLE
GUI Interaction prototype

### ISSUE THREAD

**Issue by ghost** [atomrab/virtualchers#28]
I add an "Information" button and a C# script which is handling GUI interaction for the Room 36.
You can click the button and then the GUI will "ShowUp". Or you can press "i" key on the keyboard.
All the GUI elements are customizable. It contains of Text Header, Text Body, Text Footer, Background color and an Image Objects. If you can write small description about Room 36. I will add it. If you have any header and footer with ICA logo, please send me. I'm open for any changes, just let me know. 

![gui_prototype](https://cloud.githubusercontent.com/assets/5684720/14588011/872ead9a-04b6-11e6-936b-9bb1f9e739a8.jpg)

**Comment by atomrab** [comment]
I've made a folder with blurbs and photos for the other rooms, which I shared with you directly. Let's not waste space in the info windows on ICA logos; if and when we get the model online, we'll put the branding on the website that houses it.

**Comment by atomrab** [comment]
I noticed that while the original information window outside room 36 closes when you click the "close" button, none of the other windows minimize when you do this. They're just open all the time.

**Comment by atomrab** [comment]
Having the windows open and close with "i" is a good solution -- as long as you know that's how it's supposed to work, it's very smooth (and doesn't require any fiddling with the mouse). Let's stick with that.

It did raise for me, however, the issue of controllers for tablet/touchscreen devices. I was wondering how you'd open or close info windows in a tablet environment. That made me realize that I had never tested the Android apk you sent in the fall, so I did a little research and figured out which setting I had to change to make it possible to install it. 

I now have it installed on a Samsung Galaxy tablet, and it opened just fine. But once opened, I have no idea how to move around in it -- there's no keyboard, and no gesture seem to do anything (single tap, double tap, swipe, drag, two-finger drag, etc.). This is probably not a challenge for the moment, but something to think about in the longer term -- plus I wanted to let you know, finally, that I got the apk to work!
